tensorboard/webapp/deeplink/hash.ts

```diff
@@ -23,29 +23,27 @@ const TAB = '__tab__';
 
 @Injectable()
 export class HashDeepLinker implements DeepLinkerInterface {
-  private readonly tfStorage: TfStorageElement;
-
   constructor() {
-    this.tfStorage = document.createElement('tf-storage');
-    const tfGlobals = document.createElement('tf-globals');
+    document.createElement('tf-storage');
+    document.createElement('tf-globals');
 
     // Note: `migrateLegacyURLScheme()` must be called before `setUseHash`, so
     // that tfStorage reads from the actual URL, not the fake hash for tests
     // only.
-    tfGlobals.tf_globals.setUseHash(true);
-    this.tfStorage.tf_storage.migrateLegacyURLScheme();
+    getGlobals('setUseHash')?.(true);
```

Comment:
There should be minimal or no changes to this file, as long as the polymer binary properly attaches the api to the TFGlobals object.